R/render.R

```diff
@@ -249,7 +252,20 @@ render <- function(input,
                    envir = parent.frame(),
                    run_pandoc = TRUE,
                    quiet = FALSE,
-                   encoding = "UTF-8") {
+                   encoding = "UTF-8",
+                   customize = FALSE) {
```

Comment:
what does the `customize` argument is used for ? It does not seem to be used... 🤔

Reply:
Oops that is my mistake. I'll fix it.

Reply:
fixed via 0e8d7b0